backend/src/routes.rs

```diff
@@ -0,0 +1,154 @@
+use actix_web::{
+    delete, get, post, put,
+    web::{self},
+    HttpResponse, Scope,
+};
+use subxt::ext::sp_core::H256;
+use uuid::Uuid;
+
+use crate::{
+    database::{
+        dto::{AttestationRequest, Credential, Query, UpdateAttestation},
+        querys::{
+            approve_attestation_request, can_approve_attestation, can_revoke_attestation,
+            delete_attestation_request, get_attestation_request_by_id, get_attestation_requests,
+            get_attestations_count, insert_attestation_request, revoke_attestation_request,
+            update_attestation_request,
+        },
+    },
+    error::AppError,
+    AppState,
+};
+
+#[get("/{attestation_request_id}")]
+async fn get_attestation(
+    path: web::Path<Uuid>,
+    state: web::Data<AppState>,
+) -> Result<HttpResponse, AppError> {
+    let attestation_id = path.into_inner();
+
+    let attesttation = get_attestation_request_by_id(&attestation_id, &state.db_executor).await?;
+
+    Ok(HttpResponse::Ok().json(serde_json::json!(attesttation)))
+}
+
+#[get("")]
+async fn get_attestations(
+    state: web::Data<AppState>,
+    pagination_query: web::Query<Query>,
+) -> Result<HttpResponse, AppError> {
+    let pagination = pagination_query.into_inner().into();
+
+    let content_range = get_attestations_count(&state.db_executor).await;
+
+    let attestation_requests = get_attestation_requests(pagination, &state.db_executor).await?;
+
+    let response = serde_json::json!(attestation_requests);
```

Comment:
Why use the serde_json::json! here? That seems like an overkill for simply transforming it to a json::Value?

Reply:
[35574e0](https://github.com/KILTprotocol/attester/pull/1/commits/35574e09be60b7a3fae48e44320e73c232051780)

Reply:
[35574e0](https://github.com/KILTprotocol/attester/pull/1/commits/35574e09be60b7a3fae48e44320e73c232051780)